### PR TITLE
Added killSignal user option

### PR DESCRIPTION
**Pull request by franciscocpg**
According to this issue https://github.com/codeskyblue/fswatch/issues/12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ triggers:
   cmd: go test -v
   delay: 100ms
   signal: "KILL"
+  killSignal: "SIGTERM"
```
Comment (codeskyblue) [inline README.md]: It is better to use `kill_signal`

File: fswatch.go [hunk 2]
```diff
@@ -87,6 +87,8 @@ type TriggerEvent struct {
 	delayDuration time.Duration     `yaml:"-" json:"-"`
 	Signal        string            `yaml:"signal" json:"signal"`
 	killSignal    os.Signal         `yaml:"-" json:"-"`
+	KillSignal    string            `yaml:"killSignal" json:"killSignal"`
```
Comment (codeskyblue) [inline fswatch.go]: here too, use `kill_signal` instead of `killSignal`

**Comment by codeskyblue** [comment]
I'll merge it if you fix all commented code line.

**Comment by franciscocpg** [comment]
Done

**Pull request merged**